vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -1053,6 +1053,15 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
           return { key, columnId };
         });
 
+        grid.preventContextMenu =  tryCatchWrapper(function (event) {
+            const isLeftClick = event.type === 'click';
+            const firstColumnClicked = grid.getEventContext(event).column === grid._getColumns()[0]
+            const isMultiSelectGrid = selectionMode === validSelectionModes[2];
+            const selectionColumnClicked = isMultiSelectGrid && firstColumnClicked;
+
+            return isLeftClick && selectionColumnClicked;
```

Comment:
Would this work?

```suggestion
            const { column } = grid.getEventContext(event);
            return isLeftClick && column instanceOf GridFlowSelectionColumn;
```

Reply:
I wonder if the context menu is supposed to open on selection column even on right click.

Reply:
> wondering if the context menu is intended to open when right-clicking on the selection column

Yes, this also crossed my mind. Let's discuss on daily... On one side there are no issues with it today, on the other side it would be more consistent to not show it at all (on selection column)

Reply:
> Would this work?

Yes, that works. Much nicer... thanks!